src/index.ts

```diff
@@ -10,7 +10,6 @@ import * as Uploads from './uploads';
 import * as API from './resources/index';
 import { V1 } from './resources/v1/v1';
 import { V2 } from './resources/v2/v2';
-import { Webhooks } from './resources/webhooks';
```

Comment:
also not sure why this disappeared

Reply:
Oh hmm. I'm not sure either. This was some custom code stainless helped us write a while ago to make it easier to verify webhooks. I don't think anyones using these at the moment so its probably fine. I'll fix in another release.